docs/release_notes/8.x-2.0.md

```diff
@@ -0,0 +1,51 @@
+# 8.x-2.0 Release Notes
+
+## Media File Size Schema Change
+The size value for media was too small to support large files (>2.2 GB). This release includes an update that allows values up to 16 exabytes. This requires existing sites updating their islandora/islandora to run a Drupal update (either `drush updb` or visit `/update.php` in your browser).  This may take a few minutes to run depending on how many media your site has.
+
+## Removing Gemini
+It is recommended to test this method in a staging/development environment before running in your production environment.
+1. Update islandora core (likely through a command like `composer update islandora/islandora`)
+2. Import two new Islandora views
+
+    i. This can be done by coping the views into a new directory and using drush such as 
+    ```
+    mkdir /var/www/html/drupal/config/1time
+    cp /var/www/html/drupal/web/modules/contrib/islandora/modules/islandora_core_feature/config/install/views.view.all_taxonomy_terms.yml /var/www/html/drupal/config/1time/
+    cp /var/www/html/drupal/web/modules/contrib/islandora/modules/islandora_core_feature/config/install/views.view.non_fedora_files.yml /var/www/html/drupal/config/1time/
+    drush config:import --partial --source /var/www/html/drupal/config/1time
+    ```
+    ii. It can also be done through the UI at `/admin/config/development/configuration/single/import`
+3. Update crayfish-commons for drupal with something like `cd /var/www/html/drupal && composer update islandora/crayfish-commons`
+4. Update Crayfish code (likely by traversing to your crayfish installation and doing a git pull `cd /var/www/html/Crayfish && git pull`)
+5. Run composer install for Milliner such as `cd /var/www/html/Crayfish/Milliner && composer install`
+6. Run composer install for Recast such as `cd /var/www/html/Crayfish/Recast && composer install`
+7. Update the Recast config 
+
+    i. `cd /var/www/html/Crayfish/Recast/cfg`
+    ii. open `config.yaml`
+    iii. Remove `gemini_base_url`
+    iv. Update `base_url` in `fedora_resource` section. It should contain a value something like `http://localhost:8080/fcrepo/rest`
+8. Clear your cache in Drupal. Potentially like `cd /var/www/html/drupal && drush cr`
+9. Restart apache (not 100% sure if this is necesssary) `sudo systemctl restart apache2`
+9. Test to make sure you can view existing objects in Fedora by going to the Drupal UI and clicking one of the Fedora URI links
+10. Create a new object and make sure it persists to Fedora and the Fedora URI likn resolves correctly. Make sure that no new entries get added to Gemini.
+11. If you're feeling brave, you can then remove Gemini like
```

Comment:
~~If you are feeling brave, you can then remove Gemini like~~ _Once you are confident the site is working without Gemini, you can finish removing it by_